packages/react-aria-components/src/RadioGroup.tsx

```diff
@@ -133,11 +135,23 @@ function RadioGroup(props: RadioGroupProps, ref: ForwardedRef<HTMLDivElement>) {
     defaultClassName: 'react-aria-RadioGroup'
   });
 
+  useImperativeHandle<HTMLDivElement|null, HTMLDivElement|null>(ref, () => {
+    if (domRef.current) {
+      domRef.current.focus = () => {
+        if (domRef.current) {
+          let walker = getFocusableTreeWalker(domRef.current, {tabbable: true});
```

Comment:
I think we should add an accept check to filter just radios
```
{tabbable: true, accept: (node) => node.getAttribute('type') === 'radio'}
```

Reply:
Thanks for the suggestion! [af94a00](https://github.com/adobe/react-spectrum/pull/6067/commits/af94a0039a484e0864d050317e2c5fda16b541a9) contains the update.